backend/core/src/migration/1640110170049-seed-detroit-translation-entries.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { Language } from "../shared/types/language-enum"
+
+export class seedDetroitTranslationEntries1640110170049 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const defaultTranslation = {
+      t: {
```

Comment:
This will clobber the existing translation in the database. I think you'll need to somehow merge this dictionary with the initial one that was added in the [first](https://github.com/CityOfDetroit/bloom/blob/main/backend/core/src/migration/1635889621244-seed-detroit-translation-entries.ts) migration.

Reply:
what if we just re-initialize with the original translations and the new ones?

Reply:
Right, you'll need to include all the translations in the first migration, into this new migration here. Otherwise, the existing emails won't work (like the welcome email).

Reply:
ok fixed and changed it to update!

Reply:
Sorry to go back and forth on this, but it seems like in the upstream repo, they actually query the db, then update the existing entries: https://github.com/bloom-housing/bloom/blob/dev/backend/core/src/migration/1635946663862-add-change-email-translations.ts

Could you do that instead? That will make any future detective work much easier, I think.

Reply:
oops yeah I'll fix that thanks for pointing it out

Reply:
if we're just updating I don't have to include all of the original unchanged translations right?